.github/tests/lockdown/post-install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+if [ $1 -ne 0 ]; then
+  echo '==> Events of namespace spire-server'
```

Comment:
Better to write this to GITHUB_STEP_SUMMARY like I do in my draft pr. You can format it in markdown code blocks.